### DISCUSSION TITLE
01/

### DISCUSSION THREAD

**Discussion by giscus** [livialima/linuxupskillchallenge#5] [General]
# 01/

A month-long course aimed at those who aspire to get Linux-related jobs in the industry - junior Linux sysadmin, DevOps-related work, and similar. Learn the skills required to sysadmin a remote Linux server from the commandline.

https://linuxupskillchallenge.org/01/

**Comment by HTH24** [comment]
To run "lshw", you need "sudo apt-get install lshw"

**Reply by sotenna** [reply]
Thanks

**Comment by DanielYoungDev17** [comment]
You might need to run the command below to get ifconfig to work 
 sudo apt install net-tools

**Reply by sadityakumar9211** [reply]
That's because ifconfig binary comes packaged with net-tools module which may not be installed in your distribution.

**Comment by eliseumarmon** [comment]
Notes in the manpage of netstat: This program is mostly obsolete.  Replacement for netstat is ss.
       Replacement for netstat -r is ip route.  Replacement for netstat
       -i is ip -s link.  Replacement for netstat -g is ip maddr.
